From faf0a7ced42d9a1aab76d94842fe8515e5441b03 Mon Sep 17 00:00:00 2001
From: Huang Loren <b02279@freescale.com>
Date: Wed, 22 Feb 2012 14:41:00 +0800
Subject: [PATCH 1153/2463] ENGR00175090 Update the fsl copyright of gpu
 kernel driver to 2012

Update the fsl copyright of gpu kernel driver to 2012

Signed-off-by: Huang Loren <b02279@freescale.com>
---
 .../hal/os/linux/kernel/gc_hal_kernel_driver.c     |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/mxc/gpu-viv/hal/os/linux/kernel/gc_hal_kernel_driver.c b/drivers/mxc/gpu-viv/hal/os/linux/kernel/gc_hal_kernel_driver.c
index 5db0626..496082e 100644
--- a/drivers/mxc/gpu-viv/hal/os/linux/kernel/gc_hal_kernel_driver.c
+++ b/drivers/mxc/gpu-viv/hal/os/linux/kernel/gc_hal_kernel_driver.c
@@ -1,7 +1,7 @@
 /****************************************************************************
 *
 *    Copyright (C) 2005 - 2012 by Vivante Corp.
-*    Copyright (C) 2011 Freescale Semiconductor, Inc.
+*    Copyright (C) 2011-2012 Freescale Semiconductor, Inc.
 *
 *    This program is free software; you can redistribute it and/or modify
 *    it under the terms of the GNU General Public License as published by
-- 
1.7.7.4

